api/v1alpha1/metric_types.go

```diff
@@ -72,4 +72,6 @@ type OpenTelemetrySink struct {
 }
 
 type PrometheusProvider struct {
+	// Disabled disables the Prometheus endpoint.
+	Disabled bool `json:"disabled,omitempty"`
```

Comment:
should be `disable` (in present tense)
e.g. https://github.com/envoyproxy/gateway/blob/f203fe51dbcc2b6da3d3d5ea8a1715aad0eb1a1e/api/v1alpha1/accesslogging_types.go#L10